BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankNetworkingScreenApiCoordinator.swift

```diff
@@ -151,6 +151,24 @@ open class GiniBankNetworkingScreenApiCoordinator: GiniScreenAPICoordinator, Gin
         self.trackingDelegate = trackingDelegate
     }
 
+    public init(resultsDelegate: GiniCaptureResultsDelegate,
```

Comment:
[Suggestion]: I think we can make this method private, I would keep `lib` creation under the hood

Reply:
Sure, done